packages/jest-cli/src/runGlobalHook.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+import type {GlobalConfig} from 'types/Config';
+import type {Test} from 'types/TestRunner';
+
+export default async ({
+  allTests,
+  globalConfig,
+  moduleName,
+}: {
+  allTests: Array<Test>,
+  globalConfig: GlobalConfig,
+  moduleName: string,
```

Comment:
Let's be explicit, there can only be 2 possibilities here:
```suggestion
  moduleName: 'globalSetup' | 'globalTeardown',
```